docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,196 @@
+What's New in Iris 2.3.0
+************************
+
+:Release: 2.3.0
+:Date: 2019-10-04
+
+This document explains the new/changed features of Iris in version 2.3.0
+(:doc:`View all changes <index>`.)
+
+Iris 2.3.0 Features
+===================
+.. _showcase:
+
+.. admonition:: Increased Support for CF 1.7
+
+  Several changes have contributed to our increased support for CF 1.7:
+
+  * climatological coordinate support
+  * standard name modifier support
+  * name tokens enforcement in cell methods
+  * coordinate var_name, CF_CONVENTIONS_VERSION advanced to 1.7
+
+  You can read more about each of these below.
+
+.. admonition:: Climatological Coordinate Support
+
+  Iris can now load, store and save `NetCDF climatological coordinates
+  <http://cfconventions.org/Data/cf-conventions/cf-conventions-1
+  .7/cf-conventions.html#climatological-statistics>`_. Any cube time
+  coordinate can be marked as a climatological time axis using the boolean
+  property: ``climatological``. The climatological bounds are stored in the
+  coordinate's ``bounds`` property.
+
+  When an Iris climatological coordinate is saved in NetCDF, the NetCDF
+  coordinate variable will be given a 'climatology' attribute, and the
+  contents of the
+  coordinate's ``bounds`` property are written to a NetCDF boundary variable
+  called 'climatology_bounds'.  These are in place of the more usual NetCDF
+  'bounds' attribute and accompanying 'bnds' boundary variable. See below
+  for an `example from CF conventions <http://cfconventions
+  .org/Data/cf-conventions/cf-conventions-1.7/cf-conventions
+  .html#climatological-statistics>`_:
+
+    .. code-block:: none
+
+        dimensions:
+          time=4;
+          nv=2;
+        variables:
+          float temperature(time,lat,lon);
+            temperature:long_name="surface air temperature";
+            temperature:cell_methods="time: minimum within years time: mean over years";
+            temperature:units="K";
+          double time(time);
+            time:climatology="climatology_bounds";
+            time:units="days since 1960-1-1";
+          double climatology_bounds(time,nv);
+        data:  // time coordinates translated to date/time format
+          time="1960-4-16", "1960-7-16", "1960-10-16", "1961-1-16" ;
+          climatology_bounds="1960-3-1",  "1990-6-1",
+                             "1960-6-1",  "1990-9-1",
+                             "1960-9-1",  "1990-12-1",
+                             "1960-12-1", "1991-3-1" ;
+
+  If a climatological time axis is detected when loading NetCDF -
+  indicated by the format described above - the ``climatological`` property
+  of the Iris coordinate will be set to ``True``.
+
+.. admonition:: New Chunking Strategy
+
+  Iris now makes better choices of Dask chunk sizes when loading from NetCDF
+  files: If a file variable has specified, small chunks, Iris will now choose
```

Comment:
I remember this phrasing tripping me up in the PR it was included. I think the ordering of the adjectives are wrong.
I think it should be:
`If a file variable has small, specified chunks, Iris will now choose`

Reply:
👍